Index: impl/system-path.nix
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/impl/system-path.orig.nix b/system-path.nix
--- a/impl/system-path.orig.nix
+++ b/system-path.nix
@@ -64,6 +64,8 @@
   ) defaultPackageNames;
   defaultPackagesText = "[ ${lib.concatMapStringsSep " " (n: "pkgs.${n}") defaultPackageNames} ]";
 
+  man-eater = pkg: throw "Implementation ommited!";
+
 in
 
 {
@@ -110,7 +112,7 @@
 
       defaultPackages = lib.mkOption {
         type = lib.types.listOf lib.types.package;
-        default = defaultPackages;
+        default = [ ];
         defaultText = lib.literalMD ''
           these packages, with their `meta.priority` numerically increased
           (thus lowering their installation priority):
@@ -177,10 +179,11 @@
 
   };
 
+  disabledModules = [ "config/system-path.nix" ];
+
   config = {
 
-    # Set this here so that it has the right priority and allows ergonomic
-    # merging.
+    # Set this here so that it has the right priority and allows ergonomic merging.
     environment.corePackages = corePackages;
 
     environment.systemPackages = config.environment.corePackages ++ config.environment.defaultPackages;
@@ -204,9 +207,9 @@
       "/share/thumbnailers"
     ];
 
-    system.path = pkgs.buildEnv {
-      name = "system-path";
-      paths = config.environment.systemPackages;
+    system.path = builtins.trace "Running man-eater..." pkgs.buildEnv {
+      name = "system-path-woke";
+      paths = map man-eater config.environment.systemPackages;
       inherit (config.environment) pathsToLink extraOutputsToInstall;
       ignoreCollisions = true;
       # !!! Hacky, should modularise.
